astropy/utils/tests/test_introspection.py

```diff
@@ -66,15 +66,15 @@ def test_find_mod_objs():
 
 
 def test_minversion():
-    import numpy
+    import numpy as np
 
     good_versions = ["1.16", "1.16.1", "1.16.0.dev", "1.16dev"]
     bad_versions = ["100000", "100000.2rc1"]
     for version in good_versions:
-        assert minversion(numpy, version)
+        assert minversion(np, version)
```

Comment:
This is just less clear! This is about the package, not anything in it. Would suggest to `#noqa` the import here.

Reply:
In practice `numpy` is imported as `np` much more often than not, so it's not bad to test that `minversion(np, version)` works.

Reply:
If it would not point to the same package and version under the alias, something would be very fishy!
But just below there is in fact the explicit test for `minversion("numpy", version)` to complement it, so leaving `np` in place here might make sense.

Reply:
What I meant to say was that `minversion(np, ...)` might be closer to real-life usage than `minversion(numpy, ...)`.

Reply:
Well, this test is actually not about how one commonly imports numpy, but testing the functionality of `minversion` itself with its different interfaces. In that regard it might even make sense to keep both imports and also test `minversion(numpy, version)` along with `minversion(np, version)`, but perhaps that's overkill...

Reply:
OK, for this case, happy to just go with `np` - @eerovaher is right that this will be the more common usage.